src/30_Advanced/Star_Rating.html

```diff
@@ -47,7 +47,6 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
       -webkit-tap-highlight-color: transparent;
       /* border: 1px solid red;  /* debug */
-      margin-bottom: 1em;
     }
 
     /* the stars */
```

Comment:
For this sample, the section "Usage" is rendered as if it was a sample but it's actually only instructions.
![usage](https://cloud.githubusercontent.com/assets/820891/23922216/614106a8-08f9-11e7-810c-79bcb6407742.png)